BankAPILibrary/GiniBankAPILibrary/Sources/GiniBankAPILibrary/Documents/Core/GiniBankAPI.swift

```diff
@@ -149,5 +150,12 @@ extension GiniBankAPI {
                     "Check that the Keychain capability is enabled in your project")
             }
         }
+
+        private func runningUnitTests() -> Bool {
+            #if canImport(XCTest)
+            return ProcessInfo.processInfo.environment["XCTestConfigurationFilePath"] != nil
+            #endif
+            return false
+        }
```

Comment:
Yes, we need it. There’s use of Keychain where that method is used. And during test, it crashed there for me, probably because it required an entitlement, and test targets cannot have entitlements I think